src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -6,27 +6,52 @@
                 <div id="dataset-id" class="mt-4">
                     <span v-translate="'uploadFileDataset'"></span>
                     <span>{{ dataset }}</span></div>
-                <div id="instructions" class="mt-3" v-translate="'uploadFileInstruction'"></div>
-                <div v-for="(uploadFileSection, sectionIndex) in uploadFileSections" :key="sectionIndex">
+                <div class="pt-3">
+                    <input
+                        type="radio"
+                        id="createRelease"
+                        value="createRelease"
+                        v-model="choiceUpload"
+                    />
+                    <label for="createRelease" v-translate="'createRelease'" class="pr-1"></label>
+                    <span class="icon-small" v-tooltip="translate('createReleaseTooltip')">
+                        <help-circle-icon></help-circle-icon>
+                    </span>
+                    <br />
```

Comment:
The wrapping here is a bit odd - especially in non-English language. Radio button, text and help icon all appearing on different lines?
![Screenshot from 2021-09-17 09-37-44](https://user-images.githubusercontent.com/1724545/133759074-d7c24bce-226f-431e-b5c3-05d647b61a6a.png)

Reply:
Up to you, but I found that the radio buttons rendered better if I tweaked the HTML to be more consistent with the file list (sorry this is raw HTML rather than Vue, but should be easy to translate back):
```
<div class="mt-3 form-check">
  <input type="radio" id="uploadFiles" value="uploadFiles" class="form-check-input">
  <span class="form-check-label">
    <label for="uploadFiles" class="d-inline">Créer une nouvelle version ADR (télécharger tous les fichiers)</label>
    <span class="icon-small d-inline has-tooltip" data-original-title="null">
      <svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-help-circle"><circle cx="12" cy="12" r="10"></circle><path d="M9.09 9a3 3 0 0 1 5.83 1c0 2-3 3-3 3"></path><line x1="12" y1="17" x2="12" y2="17"></line></svg>
    </span>
  </span>
</div>
```
Otherwise the button isn't vertically aligned with first line of text, and the horizontal spacing isn't consistent - see below:
![Screenshot from 2021-09-22 17-33-35](https://user-images.githubusercontent.com/1724545/134384778-145a3f90-e590-4a84-8e5c-4da3630c36d4.png)

Reply:
Personally, I don't see an issue with the current formatting. I think having the radial button vertically centred on the label looks fine and reducing the horizontal space between the two makes it look crowded. I applied the html structure you suggested above and this is what resulted, which I don't really prefer.
![image](https://user-images.githubusercontent.com/57633087/134502279-e5842108-6164-40fa-8439-72ac948f740b.png)